tests/models/config.spec.ts

```diff
@@ -0,0 +1,449 @@
+import {Config} from '../../addon/ng2/models/config';
+
+const fs      = require('fs');
+const path    = require('path');
+const expect  = require('chai').expect;
+const schema  = path.resolve(process.cwd(), 'test.json');
```

Comment:
It's not there as it is not needed it is [created](https://github.com/jkuri/angular-cli/blob/ng-config/tests/models/config.spec.ts#L18) before each test is run and removed [after](https://github.com/jkuri/angular-cli/blob/ng-config/tests/models/config.spec.ts#L24) test is done.